### ISSUE TITLE
Broken link to AInspector

### ISSUE THREAD

**Issue by pczarn2** [opena11y/ainspector-for-firefox#59]
Good morning, on this page, the link to download AInspector is broken:
https://ainspector.github.io/

Also, the information on this page mentions WCAG 2.0, but there is no mention of support for WCAG 2.1 or 2.2. which AInspector supports.

**Comment by jongund** [comment]
@pczarn2 
Thank you for reporting this issue, I need to get this page updated.  It is an old page that we do not use any more, but it should be updated to point to the latest information.

**Comment by jongund** [comment]
I have updated the link to the addon:
https://ainspector.github.io/
